.github/workflows/atlantis-image.yml

```diff
@@ -45,6 +45,11 @@ jobs:
     needs: [changes]
     if: needs.changes.outputs.should-run-build == 'true'
     name: Build Image
+    permissions:
+      id-token: write
+      packages: write
+      contents: read
+      attestations: write
```

Comment:
This looks way too open. Is this the minimum possible permissions for this to work?

Reply:
It needs `write` on `id-token` (to mint an OIDC token with which it gets a signing cert) and `attestations`. I think I can get away with dropping `packages` and `contents`. Will test.